### DISCUSSION TITLE
ShowInstallationWelcome is closing apps without prompting user

### DISCUSSION THREAD

**Discussion by tmeltdowner** [PSAppDeployToolkit/PSAppDeployToolkit#668] [Q&A] [answered]
Any ideas why that might be?

Here is the command:
 Show-InstallationWelcome -CloseApps 'iexplore' -CheckDiskSpace -PersistPrompt

Here is what is logged:
[Initialization] :: Force close application(s) [Internet Explorer] without prompting user.

**Comment by mjr4077au** [comment, marked as answer]
Your DeployMode was likely silent, which would have caused this.
